services/ui-src/src/components/reports/ReportPageIntro.tsx

```diff
@@ -27,13 +26,18 @@ export const ReportPageIntro = ({
         {initiativeName ? initiativeName : subsection}
       </Heading>
       {hint && <Box sx={sx.hintTextBox}>{hint}</Box>}
-      {accordion && <InstructionsAccordion verbiage={accordion} />}
-      {stepType === "evaluationPlan" && report?.reportType === ReportType.SAR && (
+      {accordion && stepType !== "objectiveProgress" && (
```

Comment:
nit: Could swap the hard string call of objectiveProgress to OverlayModalStepTypes.OBJECTIVES_PROGRESS here and on line 32 and 38  below